bundle.Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Use OpenShift golang builder image
```

Comment:
I'm a bit confused actually. With this PR checked out I'm seeing :

```
greg@bahia:[generate-files-in-dockerfile]sandboxed-containers-operator$ git grep bundle.\*Dockerfile
Makefile:       docker build -f bundle-custom.Dockerfile -t $(BUNDLE_IMG) .
Makefile:       $(RM) bundle.Dockerfile
```

Is `bundle.Dockerfile` being used at all ?

Reply:
It is only being used by Prow CI job.  We will point it to bundle-custom.Dockerfile and remove bundle.Dockerfile after that.